src/main/scala/firrtl/Utils.scala

```diff
@@ -51,7 +51,17 @@ object getWidth {
   def apply(e: Expression): Width = apply(e.tpe)
 }
 
+/**
+ * Helper object for computing the width of a firrtl type.
+ */
 object bitWidth {
```

Comment:
This is great! Thanks for discovering this function. I did not know that it existed.

Reply:
Real credit goes to @jackkoenig  :)